package-lock.json

```diff
@@ -1,12482 +1,8 @@
 {
   "name": "@adobe/reactor-turbine",
-  "version": "27.1.1",
-  "lockfileVersion": 2,
+  "version": "27.1.3",
+  "lockfileVersion": 1,
```

Comment:
You may want to update your npm. It looks like whoever changed this last was using a never version, so you'll have unnecessary git thrashing as you take turns committing.